### ISSUE TITLE
GED Dataset

### ISSUE THREAD

**Issue by ufukcbicici** [One1h/ElDet#5]
Hello everyone. Is it possible for people outside China to download the GED dataset? I have difficulties getting it due to the Baidu hosting, I am trying to reach the data from Europe.

Best regards.

**Comment by taeraemon** [comment]
I'm suffering from the same problem. I think I can share it with Google Drive or GitHub because it's a small capacity.

**Comment by ufukcbicici** [comment]
Hello @taeraemon, do you have the dataset already yet?

**Comment by taeraemon** [comment]
@ufukcbicici No, I'm still struggling. I'm trying to reach data from south korea. I'm trying to ask help from reddit user who can reach to baidu netdisk.

**Comment by D49ll** [comment]
Did you ever get the Dataset?

**Comment by One1h** [comment]
The dataset has added a Google Drive download link.
